packages/common/src/UniswapV3Helpers.js

```diff
@@ -22,8 +22,8 @@ function decodePriceSqrt(priseSqrt) {
 }
 
 // Fetch the decoded price from a uniswap pool from slot0.
-async function getCurrentPrice(poolAddress) {
-  const pool = await createContractObjectFromJson(UniswapV3Pool).at(poolAddress);
+async function getCurrentPrice(poolAddress, web3) {
```

Comment:
This should be `Web3` right, since you import that on line 3?

Reply:
no sir. this is web3 ass passed in from the caller of this method.